Makefile

```diff
@@ -312,7 +312,7 @@ delete_all_virsh_resources: destroy_all_nodes delete_minikube kill_all_port_forw
 	skipper run $(SKIPPER_PARAMS) 'discovery-infra/delete_nodes.py -ns $(NAMESPACE) -a'
 
 _download_logs:
-	discovery-infra/download_logs.py $(REMOTE_SERVICE_URL) $(LOGS_DEST) --cluster-id $(CLUSTER_ID)
+	discovery-infra/download_logs.py $(REMOTE_SERVICE_URL) $(LOGS_DEST) --cluster-id $(CLUSTER_ID) --must-gather
```

Comment:
must-gather shouldn't always be provided. Only for `download_all_logs`

Reply:
now download_logs  collects only the failed logs with must-gather
so this job is not supposed to change.

added a second pipeline that collects all logs that will not collect must-gather.

Reply:
I would prefer a way to quickly download logs without must-gather.
What do you say about:

```makefile
_download_logs:
	discovery-infra/download_logs.py $(REMOTE_SERVICE_URL) $(LOGS_DEST) --cluster-id $(CLUSTER_ID) $(ADDITIONAL_PARAMS)

download_logs:
	skipper make $(SKIPPER_PARAMS) _download_logs $(ADDITIONAL_PARAMS)

download_all_logs:
	skipper make $(SKIPPER_PARAMS) _download_all_logs ADDITIONAL_PARAMS="'--download-all --must-gather  ${ADDITIONAL_PARAMS}'"
```